test/Microsoft.AspNetCore.DataProtection.Redis.Test/DataProtectionRedisTests.cs

```diff
@@ -1,17 +1,29 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNetCore.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Moq;
+using StackExchange.Redis;
+using System;
 using System.Linq;
+using System.Threading.Tasks;
 using System.Xml;
 using System.Xml.Linq;
-using Moq;
-using StackExchange.Redis;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Microsoft.AspNetCore.DataProtection
 {
     public class DataProtectionRedisTests
```

Comment:
Use LoggedTest

Reply:
Where is that typed defined?

Reply:
https://github.com/aspnet/Logging/blob/master/src/Microsoft.Extensions.Logging.Testing/LoggedTest/LoggedTest.cs

Reply:
Oh, that's not currently used in this project. What problem does that solve?

Reply:
You get your tests logs written into a global text log by default.

Reply:
Nah. This test isn't producing a substantial amount of logs. It's only produce one line of output as a debug helper to make sure your config settings are flowing through.